### PR TITLE
PXP-638: [CLI] Reached API limits on Google Logging

### PR DESCRIPTION
**Pull request by jk-gan**
## Summary

Ticket: [PXP-638](https://mindvalley.atlassian.net/browse/PXP-638)

### Changed
- [x] now the logs panel will fetch the latest 100 log entries on the first API call
- [x] temporary remove the time filter feature

[PXP-638]: https://mindvalley.atlassian.net/browse/PXP-638?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/182#" title="Author self-approved">jk-gan</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: cli/src/commands/tui/events/network.rs
Comment (mfauzaan) [inline cli/src/commands/tui/events/network.rs:428]: I suggest simplifying the code by consolidating it into a single `fetch_log_entries` function. We can modify the parameters based on tailing to enhance readability.

```suggestion
                    let mut log = match fetch_log_entries(
                        Some(resource_names.clone()),
                        Some(max_entries),
                        Some(filter.clone()),
                        None,
                        order_by,
                        gcloud_access_token.clone(),
                        wk_client,
                    )
                    .await
                    {
                        Ok(data) => data,
                        Err(error) => {
                            let mut app_ref = app.lock().await;
                            app_ref.state.log_entries_error = Some(format!("{error}"));
                            return Err(error);
                        }
                    };

                    if tailing {
                        log.entries = log
                            .entries
                            .map(|entries| entries.into_iter().rev().collect::<Vec<LogEntry>>());
                    }

```

**Review by mfauzaan — CHANGES_REQUESTED** [review]

**Review by mfauzaan — APPROVED** [review]
LGTM!

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/182#" title="Author self-approved">jk-gan</a>*, *<a href="https://github.com/mindvalley/wukong-cli/pull/182#pullrequestreview-1796119625" title="Approved">mfauzaan</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
